common/tests/fixtures/cardano/sign_tx.json

```diff
@@ -1060,7 +1060,150 @@
         "auxiliary_data_supplement": {
           "type": 1,
           "auxiliary_data_hash": "a943e9166f1bb6d767b175384d3bd7d23645170df36fc1861fbf344135d8e120",
-          "catalyst_signature": "74f27d877bbb4a5fc4f7c56869905c11f70bad0af3de24b23afaa1d024e750930f434ecc4b73e5d1723c2cb8548e8bf6098ac876487b3a6ed0891cb76994d409"
+          "governance_signature": "74f27d877bbb4a5fc4f7c56869905c11f70bad0af3de24b23afaa1d024e750930f434ecc4b73e5d1723c2cb8548e8bf6098ac876487b3a6ed0891cb76994d409"
+        }
+      }
+    },
+    {
+      "description": "transaction with CIP36 governance registration and voting purpose not specified",
+      "parameters": {
+        "protocol_magic": 764824073,
+        "network_id": 1,
+        "fee": 42,
+        "ttl": 10,
+        "validity_interval_start": null,
+        "certificates": [],
+        "withdrawals": [],
+        "auxiliary_data": {
+          "governance_registration_parameters": {
+            "staking_path": "m/1852'/1815'/0'/2/0",
+            "reward_address_parameters": {
+              "addressType": 0,
+              "path": "m/1852'/1815'/0'/0/0",
+              "stakingPath": "m/1852'/1815'/0'/2/0"
+            },
+            "nonce": 22634813,
+            "format": 1,
```

Comment:
minor nit: you should be using `"CIP36"` string

Reply:
How so? The parser expects an int enum value, just like with output `format`. Or did you mean changing the parser to expect a string? (In that case, should I also change the output `format` parser?)

Reply:
my bad, I forgot that this is a custom parser and not the built-in json-to-protobuf. no reason to muck about with it then